moveit_planners/trajopt/src/trajopt_planner_manager.cpp

```diff
@@ -37,104 +37,101 @@
 */
 
 #include <moveit/planning_interface/planning_interface.h>
-
 #include <moveit/collision_detection_fcl/collision_detector_allocator_fcl.h>
-
 #include <class_loader/class_loader.hpp>
-
 #include <trajopt_interface/trajopt_planning_context.h>
 
 namespace trajopt_interface
 {
-class TrajOptPlannerManager : public planning_interface::PlannerManager
-{
-public:
-  TrajOptPlannerManager() : planning_interface::PlannerManager()
-  {
-  }
-
-  bool initialize(const moveit::core::RobotModelConstPtr& model, const std::string& ns) override
+  class TrajOptPlannerManager : public planning_interface::PlannerManager
```

Comment:
When CI runs, I think the `clang format` check will fail. It automatically formats the indentation so it's consistent across all MoveIt repositories. So this change will probably need to be reverted.